source/agora/consensus/EnrollmentManager.d

```diff
@@ -49,6 +50,9 @@ public class EnrollmentManager
     /// Node's key pair
     private Pair key_pair;
 
+    /// Key used for enrollment which is actually an UTXO hash
+    private Hash enroll_key;
```

Comment:
If it's "actually an UTXO hash", why is it named key then ?

Reply:
Because It's used in order to select an enrollment information by the key, or `enroll_key` in the validator set table. What I intended to represent by the name is that it's the key for an enrollment information.